assets/js/blocks/cart/edit.js

```diff
@@ -101,9 +101,12 @@ export const Edit = ( { className, attributes, setAttributes, clientId } ) => {
 					currentView={ currentView }
 					previewData={ { previewCart } }
 				>
-					<BlockControls __experimentalShareWithChildBlocks>
-						{ ViewSwitcherComponent }
-					</BlockControls>
+					<BlockSettings
+						attributes={ attributes }
+						setAttributes={ setAttributes }
+					/>
```

Comment:
Hi @tarunvijwani! The `BlockSettings` component is already added on line 80 of this file. With the current two `BlockSettings` we have double dark mode controls:

<img width="307" alt="image" src="https://user-images.githubusercontent.com/14235870/186941705-2d2a5ec8-1f88-4e31-afb5-d4eb985a1a08.png">

Reply:
Sorry, I missed them while resolving the merge conflict. I have removed the duplicate code. Thank you!